server.js

```diff
@@ -22,6 +22,13 @@ const manager = api.manager();
 const port = process.env.SPLITIO_SERVER_PORT || 7548;
 const EXT_API_KEY = process.env.SPLITIO_EXT_API_KEY;
 
+// Healthcheck used by kubernetes does not check api key
+app.get('/', (req, res) => {
```

Comment:
Thanks for sending this. Does it have to be `/` or can it be something like `/status` ?

Reply:
I was wondering if you would be willing to rename this to `/admin/ping`

```
/admin/ping
A ping endpoint to monitor the service status. If the service is up and running it will respond with "pong" and a 200 HTTP status code.
```

Mostly to keep consistency with: https://docs.split.io/docs/split-synchronizer#section-admin-tools-proxy which is another service we offer.

Reply:
@rmelick-vida

Reply:
I think the healthcehck can be changed: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-http-request

Reply:
@patricioe Looking further into it, it seems like I might be able to change to a different path.  I am creating the load balancer as a Kubernetes Ingress, and I was not setting up the readiness probe for my pod at all, it was defaulting to /.  I just found this doc today though: https://github.com/kubernetes/ingress-gce/blob/master/README.md#health-checks, so it seems like a possibility.  I will try the new approach with /admin/ping today.  Hopefully it works and I'll update the PR :).